content/3_Silver/2P.mdx

```diff
@@ -31,6 +31,143 @@ frequency: 2
 	</Resource>
 </Resources>
 
+## Two Pointers
+
+The two pointers method iterates two pointers across an array, to track the start and end
+of an interval. It can also be used to track two values in order to satisfy a specific condition. 
```

Comment:
still sorta unclear. To a new reader with no knowledge of 2p, "used to trace two values in order to satisfy a specific condition" is simply too broad and isn't very helpful.